client/src/js/filters/bytes.js

```diff
@@ -0,0 +1,23 @@
+angular.module('bhima.filters')
+.filter('bytes', BytesFilter);
+
+BytesFilter.$inject = ['$translate'];
+
+/**
+ * Bytes Filter
+ * Format bytes size to human readable format
+ */
+function BytesFilter($translate) {
+	return function(bytes, precision) {
+		if (isNaN(parseFloat(bytes)) || !isFinite(bytes)) { return '-'; }
+		if (typeof precision === 'undefined') { precision = 1; }
+		var units = ['SIZE_BYTES', 'SIZE_KB', 'SIZE_MB', 'SIZE_GB', 'SIZE_TB', 'SIZE_TB'],
```

Comment:
Cool!  Another optimization (not necessary for this PR), would be to move the keys and `$translate` call out of the `return` statement, since they never change.

Like:
```js
function BytesFilter($translate) {
  var units = ['SIZE_BYTES', 'SIZE_KB', 'SIZE_MB', 'SIZE_GB', 'SIZE_TB', 'SIZE_TB'];
  units = units.map(function (u) { return 'FORM.LABELS.' + $translate.instant(u); });
  return function (bytes, precision) { 
   // code ....
  };
}
```

Just an idea. :)